CMakeLists.txt

```diff
@@ -374,7 +373,7 @@ add_library(transformer-shared SHARED
 
 if (BUILD_MULTI_GPU)
 target_link_libraries(transformer-shared PUBLIC
-  -lmpi
+  ${MPI_CXX_LIBRARIES}
```

Comment:
How is this related to this PR?

Reply:
Not related. I'll remove it and pull another PR